intTests/test_llvm_bound_check/Makefile

```
@@ -0,0 +1,2 @@
+bound.bc : bound.c
+	clang -c -emit-llvm -o bound.bc bound.c
```

Slightly terser:

```suggestion
bound.bc : bound.c
	clang -c -emit-llvm -o $@ $<
```

Also, consider adding `-frecord-command-line` as an option.